americangut/make_phyla_plots.py

```diff
@@ -14,6 +13,9 @@
 from operator import itemgetter
 import colorbrewer
 
+
+use('agg')
```

Comment:
Ok, I'll give it a shot and see what Travis says. Fwiw, my local flake8
wasn't complaining about the imports not being at the top of the file but
Travis was, must be different versions or configuration
On Mar 26, 2015 6:56 PM, "Yoshiki Vázquez Baeza" <notifications@github.com>
wrote:

> In americangut/make_phyla_plots.py
> <https://github.com/biocore/American-Gut/pull/139#discussion_r27269991>:
>
> > @@ -14,6 +13,9 @@
> >  from operator import itemgetter
> >  import colorbrewer
> >
> > +
> > +use('agg')
>
> Matplotlib recommends that this be set before any package makes an import
> from pyplot: http://matplotlib.org/faq/usage_faq.html#what-is-a-backend
> However I'm not sure if this is a problem with this package, thus I
> suggested the noqa thing just to avoid the possibility of it being a
> problem.
> … <#14c58bb27e823bea_>
> On (Mar-26-15|17:31), adamrp wrote: > @@ -14,6 +13,9 @@ > from operator
> import itemgetter > import colorbrewer > > + > +use('agg') It was
> complaining about the other lines (imports not being at top of file); will
> the `noqa` directive fix those complaints? Also, does the placement of this
> call matter with respect to the imports? --- Reply to this email directly
> or view it on GitHub:
> https://github.com/biocore/American-Gut/pull/139/files#r27269055
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/American-Gut/pull/139/files#r27269991>.
>